app/views/docs/functions.phtml

```diff
@@ -93,25 +93,54 @@ $runtimes = $this->getParam('runtimes', []);
     <p><b>Python</b></p>
 
     <div class="ide margin-bottom" data-lang="bash" data-lang-label="Bash">
-        <pre class="line-numbers"><code class="prism language-bash" data-prism>PIP_TARGET=./.appwrite pip install -r ./requirements.txt --upgrade --ignore-installed</code></pre>
```

Comment:
I'm almost certain that this is valid and correct. Is there a reason you think the single line command throws an error ? 

Let's keep it as a single line command as long as there is no good reason to change it to 2 lines.

Reply:
you can keep it as a single command if you want.

but 2 points:
first of all settings env var is not part of the command so it should be separated.
if you still want it as a single command, thre should be a ":" betwen the env var and the command.

Reply:
moved it all to 1 liner.

**I believe it should be split since env var is important all the time and not per-command.
but its your show :)